packages/notebook/src/browser/view/notebook-code-cell-view.tsx

```diff
@@ -148,37 +149,52 @@ interface NotebookCellOutputProps {
 
 export class NotebookCodeCellOutputs extends React.Component<NotebookCellOutputProps> {
 
-    protected outputsWebview: CellOutputWebview | undefined;
+    protected outputsWebview: CellOutputWebview | Promise<CellOutputWebview> | undefined;
```

Comment:
Suggestion: I feel like this leads to some anti-patterns. See in line 165:

```ts
this.outputsWebview = outputWebviewFactory(cell).then(webview => {
    this.outputsWebview = webview;
    ...
```
We should split this into a `outputsWebview` and `outputsWebviewPromise`.